commons/helpers/pods_helper.py

```diff
@@ -526,3 +513,30 @@ def get_all_cluster_processes(self, pod_name, container_name):
                                  decode=True)
         process_list = resp.splitlines()
         return process_list
+
+    def get_deployment_name(self, pod_prefix=None):
```

Comment:
Can you check if get_deploy_replicaset() can satisfy your requirement.

Reply:
This seems to be cleaner way to retrieve deployment name and this is been used in CSM tests as well.